src/ui/LoadImage.js

```diff
@@ -29,13 +29,15 @@ function LoadImage(ref, name, src, main_callback) {
       var image = document.createElement('img');
       var canvas = document.createElement('canvas');
       var context = canvas.getContext('2d');
+      
       image.onload = function() {
         canvas.width = image.naturalWidth;
         canvas.height = image.naturalHeight;
         context.drawImage(image, 0, 0);
         datauri = canvas.toDataURL(ext);
         callback(datauri, step);
       };
+      image.crossOrigin = 'Anonymous';
```

Comment:
Is this the same cross-origin flag which fixes CORS issues for servers? If that is the case then isn't wildcard `*` a better solution?

Reply:
No, it fix "operation is insecure" error when we trying to draw image on canvas (toDataUrl()). The special CORS headers from server are still required

Reply:
I don't know much about this. I'll do some research.

Reply:
What should I do with my cgi tasks? I cant submit another task until this one is not accepted

Reply:
I am not a mentor. I am also one of the students. Maybe one of @publiclab/is-reviewers can help you.

Reply:
@SidharthBansal